### DISCUSSION TITLE
Include API file which shows overview of all tracked sites

### DISCUSSION THREAD

**Discussion by LukePrior** [upptime/upptime#414] [Ideas]
I am tracking ~100 sites and I like the ability to create Shields.io badges for them but it would be really useful to have an average or overall file which shows the average uptime and response time across all the monitored sites.

**Comment by LukePrior** [comment]
I see that summary.json exists but this doesn't contain an entry for the overall average so I can't create a Badges.io shield from it.

**Comment by MatadorProBr** [comment]
You can use the ``api`` directory to make a shield in Shields.io, for more information: https://upptime.js.org/docs/badges
